renderer/src/App.js

```diff
@@ -31,13 +31,19 @@ class App extends Component {
       name: item.name,
       commit: item.commit,
     }));
-
+    const catalogsInfo = {
+      'name': tuture.name,
+      'language': tuture.language,
+      'maintainer': tuture.maintainer,
+      'topics': tuture.topics,
+    };
```

Comment:
Here you can use ES6 Destructuring: `const { a, b, c } = obj;`

At this time, you maybe need:

```javascript
  const { name, language, maintainer, topics } = tuture;
```

It's easy, uh! 😁